lib/get-ajv-error-messages.js

```diff
@@ -42,12 +42,33 @@ const getDetailsPerAjvKeyword = {
  * @returns {String} A human-readable validation error message.
  */
 module.exports = function getAjvErrorMessages(ajvErrors, rootName = `root`) {
+  console.log(ajvErrors);
   const errors = ajvErrors.filter(error => !(`propertyName` in error));
 
   return errors.map(error => {
     const getDetails = getDetailsPerAjvKeyword[error.keyword] || (() => error.message);
     const details = getDetails(error, ajvErrors);
-    const errorMessage = `${rootName}${error.dataPath} ${details}`;
-    return errorMessage.replace(`\n`, `\\n`);
+    const errorMessage = `${rootName}${error.dataPath}${getDataDescription(error.data)} ${details}`;
+    return errorMessage.replace(/\n/g, `\\n`);
   });
 };
+
+/**
+ * @param {*} data Any kind of data; not all types can be represented.
+ * @returns {String} A short representation of the given data with a leading space. Empty if this is not possible.
+ */
+function getDataDescription(data) {
+  if (data.constructor === String) {
```

Comment:
In JS, `typeof "foo"` is `string` whereas `typeof new String("foo")` is `object`. We probably won't have to handle String objects, but the constructor check is a bit safer. JS is weird, indeed.

Reply:
See https://stackoverflow.com/a/54262665 and https://stackoverflow.com/a/32420242 (lodash does indeed have a more complicated check, but uses the simple `typeof` to short-circuit it). I thing just `typeof` works fine for us.

Reply:
Also, it's a defined scope, the parameter will always be an output from AJV, so it should always work (since they don't use `new String`:

![Screenshot_2020-03-27_14-20-03](https://user-images.githubusercontent.com/202916/77760070-1587fd00-7036-11ea-8b5e-e7d35e6c7eee.png)